backend/services/music_gen_service/midi.py

```
@@ -0,0 +1,553 @@
+        key: String of key (e.g. "C", "Db", "G#")
+        octave: Integer of octave to use (default is 4)
+        
+    Returns:
+        List of MIDI note values
+    """
+    chord = harmony.chord.Chord(chord_name)
+    return [note.midi for note in chord.pitchedCommonName]
+
+
+
+def _convert_duration_to_beats(duration_str, time_signature: List[int] = [4, 4]) -> float:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)